packages/rum/src/boot/recorderApi.spec.ts

```diff
@@ -23,6 +23,25 @@ describe('makeRecorderApi', () => {
   let rumInit: (options?: { worker?: DeflateWorker }) => void
   let startSessionReplayRecordingManually: boolean
 
+  function setupRecorderApi(sessionManager: RumSessionManager = createRumSessionManagerMock().setId('1234')) {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏The test setup is splitted between `beforeEach` and `setupRecorderApi` without clear separation of concern. To simplify you could move all the test setup in `setupRecorderApi`